server/szurubooru/func/util.py

```diff
@@ -41,6 +41,16 @@ def create_temp_file(**kwargs: Any) -> Generator:
         os.remove(path)
 
 
+@contextmanager
+def create_temp_file_path(**kwargs: Any) -> Generator:
+    (descriptor, path) = tempfile.mkstemp(**kwargs)
+    os.close(descriptor)
```

Comment:
Hey that's pretty cool function.

Reply:
Ah, well I can't take credit for that. It's 90% of the function above it, I just needed the file path instead of an open file descriptor.